charts/enforcer-k8s/values.yaml

```diff
@@ -41,7 +41,30 @@ affinity: {}
 
 rode:
   host: rode:50051
-  insecure: false
+
+  ## disableTransportSecurity will disable GRPC transport security when set to `true`
+  ##
+  disableTransportSecurity: false
+
+  ## Rode authentication settings
+  ## Use rode.auth.oidc.enabled to enable OIDC auth
+  ## Use rode.auth.basic.enabled to enable basic auth
+  ##
+  auth:
+    oidc:
```

Comment:
I think we'll want to have an option for scopes here as well, it can be in a followup.